website/routes.py

```diff
@@ -504,8 +504,8 @@ def make_url_map(app):
                 '/registeredreport/',
             ],
             'get',
-            registries_views.registered_reports_landing,
-            OsfWebRenderer('registered_reports_landing.mako', trust=False)
+            website_views.redirect_to_registration_workflow_from_rr,
```

Comment:
Why is this getting its own redirect instead of getting lumped in with the `/prereg/` rule below? Or at least just reusing the same function (since the new function is purely a wrapper around it)?

Reply:
It's a quirk of Flask, but the functions must have unique names, I tried what you suggested initially and it it failed many tests.

Reply:
Makes sense that each rule might need its own callback, but why can't it get lumped into the existing rule for `redirect_to_registration_workflow`?

Reply:
Oh great point, I just didn't think to do that, it will just need a comment explaining they are legacy routes.